devel/glib2-upstream/Portfile

```diff
@@ -161,6 +161,11 @@ post-destroot {
 
 platform darwin {
     configure.args-append   -Ddtrace=false
+    
+    # some tests fail to build on 10.6.8 Rosetta: https://trac.macports.org/ticket/65517
+    if {${os.major} == 10 && ${build_arch} eq "ppc"} {
```

Comment:
Would this also apply to 10.4 and 10.5, such that we'd update the check to `${os.major} <= 10`? Or is this truly specific to 10.6?

CC: @evanmiller

Reply:
> Would this also apply to 10.4 and 10.5, such that we'd update the check to `${os.major} <= 10`? Or is this truly specific to 10.6?

@mascguy I cannot test at the moment on anything but Rosetta, but I will check my portfiles if I used to disable tests or not (that won’t guarantee anything though).

Reply:
UPD. @mascguy So judging by my portfiles for `glib` 2.72.1 from Leopard and 10.6 PPC, I did not disable tests there, which I did for Rosetta. This likely implies that tests build fine on PPC natively, and only Rosetta has an issue. (Unless of course something changed in tests since then.)

P. S. By the way, while I will have trouble here pushing any 10.6 PPC-specific fixes, if a fix is actually specific to Rosetta (and to 10.6.8 by implication), and not needed for 10.6 PPC, can it be made more precise? So that it is applied only for 10.6.8 and `ppc` arch. Not a big deal, but just in case.

Reply:
> UPD. @mascguy So judging by my portfiles for `glib` 2.72.1 from Leopard and 10.6 PPC, I did not disable tests there, which I did for Rosetta. This likely implies that tests build fine on PPC natively, and only Rosetta has an issue. (Unless of course something changed in tests since then.)
> 
> P. S. By the way, while I will have trouble here pushing any 10.6 PPC-specific fixes, if a fix is actually specific to Rosetta (and to 10.6.8 by implication), and not needed for 10.6 PPC, can it be made more precise? So that it is applied only for 10.6.8 and `ppc` arch. Not a big deal, but just in case.

I'm fine with your changes as-is, since they only apply to 10.6 PPC.